x-pack/plugins/reporting/public/components/report_listing.tsx

```diff
@@ -274,6 +337,39 @@ export class ReportListing extends Component<Props, State> {
     }
 
     if (this.mounted) {
+      const { intl } = this.props;
+      const getStatusLabel = (statusString: string) => {
+        switch (statusString) {
+          case JobStatuses.JOB_STATUS_PENDING:
+            return intl.formatMessage({
+              id: 'xpack.reporting.JobStatuses.pendingText',
```

Comment:
why `JobStatuses` instead of `listing` ? the same for other ids. I you need 'JobStatuses' namespace, it should start with lowercase then